### DISCUSSION TITLE
Sensors do not report some information on ZBINFO command

### DISCUSSION THREAD

**Discussion by rp2019433** [arendst/Tasmota#21956] [Ideas]
### PROBLEM DESCRIPTION
When issuing zbinfo <sensor> for some kind of sensors, specific attributes are not shown.

**_FOR AQARA MOTION SENSOR:_**

TELEMETRY - **Occupancy** is on the output

```
09:36:59.029 RSL: SENSOR = {"ZbInfo":{" Sensor de Movimento":{"Device":"0x1F83","Name":" Sensor de Movimento","IEEEAddr":"0x00258F010AB77CB5","ModelId":"lumi.sensor_motion.aq2","Manufacturer":"LUMI","Endpoints":[1],"Config":["I01"],"Illuminance":11,"Occupancy":0,"Reachable":true,"BatteryPercentage":100,"BatteryLastSeenEpoch":1723551657,"LastSeen":925,"LastSeenEpoch":1723551694,"LinkQuality":87}}}
```

ZBINFO - **Occupancy** is on the output
```
09:36:59.022 CMD: zbinfo 0x1F83
09:36:59.029 RSL: SENSOR = {"ZbInfo":{" Sensor de Movimento":{"Device":"0x1F83","Name":" Sensor de Movimento","IEEEAddr":"0x00258F010AB77CB5","ModelId":"lumi.sensor_motion.aq2","Manufacturer":"LUMI","Endpoints":[1],"Config":["I01"],"Illuminance":11,"Occupancy":0,"Reachable":true,"BatteryPercentage":100,"BatteryLastSeenEpoch":1723551657,"LastSeen":925,"LastSeenEpoch":1723551694,"LinkQuality":87}}}
09:36:59.035 RSL: RESULT = {"ZbInfo":"Done"}
```

**_FOR AQARA DOOR SENSOR:_**

TELEMETRY - **Contact** is on the output

```
09:30:31.304 RSL: SENSOR = {"ZbReceived":{"Sensor de Chuva":{"Device":"0xF288","Name":"Sensor de Chuva","Contact":0,"Endpoint":1,"LinkQuality":214}}}
09:30:31.404 RSL: SENSOR = {"ZbReceived":{"Sensor de Chuva":{"Device":"0xF288","Name":"Sensor de Chuva","Contact":1,"Endpoint":1,"LinkQuality":222}}}
```

ZBINFO - **Contact** is NOT on the output
```
09:31:17.918 CMD: zbinfo 0xF288
09:31:17.924 RSL: SENSOR = {"ZbInfo":{"Sensor de Chuva":{"Device":"0xF288","Name":"Sensor de Chuva","IEEEAddr":"0x00158D040AF87B24","ModelId":"lumi.sensor_magnet.aq2","Manufacturer":"LUMI","Endpoints":[1],"Config":["A01"],"ZoneStatus":12289,"Reachable":true,"BatteryPercentage":100,"BatteryLastSeenEpoch":1723550213,"LastSeen":45,"LastSeenEpoch":1723552232,"LinkQuality":218}}}
09:31:17.931 RSL: RESULT = {"ZbInfo":"Done"}
```

### REQUESTED INFORMATION
_Make sure your have performed every step and checked the applicable boxes before submitting your issue. Thank you!_

- [X] Read the [Contributing Guide and Policy](https://github.com/arendst/Tasmota/blob/development/CONTRIBUTING.md) and [the Code of Conduct](https://github.com/arendst/Tasmota/blob/development/CODE_OF_CONDUCT.md)
- [X] Searched the problem in [issues](https://github.com/arendst/Tasmota/issues)
- [X] Searched the problem in [discussions](https://github.com/arendst/Tasmota/discussions)
- [X] Searched the problem in the [docs](https://tasmota.github.io/docs/FAQ)
- [X] Searched the problem in the [chat](https://discord.gg/Ks2Kzd4)
- [X] Device used: Aqara Door Sensor (MCCGQ11LM)
- [ ] Tasmota binary firmware version number used: 14.1.0.4 (5bac2a2-zbbrdgpro)
  - [X] Pre-compiled
  - [ ] Self-compiled
- [ ] Flashing tools used: _____
- [ ] Provide the output of command: `Backlog Template; Module; GPIO 255`:
```lua
  Configuration output here:


```
- [ ] If using rules, provide the output of this command: `Backlog Rule1; Rule2; Rule3`:
```lua
  Rules output here:


```
- [X] Provide the output of this command: `Status 0`:
```
11:07:23.499 CMD: STATUS 0 
11:07:23.503 RSL: STATUS = {"Status":{"Module":0,"DeviceName":"Sonoff Zigbee Pro","FriendlyName":["Sonoff Zigbee Pro"],"Topic":"tasmota_EAEE50","ButtonTopic":"0","Power":"1","PowerLock":"0","PowerOnState":3,"LedState":1,"LedMask":"FFFF","SaveData":1,"SaveState":1,"SwitchTopic":"0","SwitchMode":[0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0],"ButtonRetain":0,"SwitchRetain":0,"SensorRetain":0,"PowerRetain":0,"InfoRetain":0,"StateRetain":0,"StatusRetain":0}}
11:07:23.509 RSL: STATUS1 = {"StatusPRM":{"Baudrate":115200,"SerialConfig":"8N1","GroupTopic":"tasmotas","OtaUrl":"https://ota.tasmota.com/tasmota32/tasmota32-zbbrdgpro.bin","RestartReason":"Software reset CPU","Uptime":"0T14:19:27","StartupUTC":"2024-08-12T23:47:56","Sleep":50,"CfgHolder":4617,"BootCount":13,"BCResetTime":"2023-12-09T13:37:40","SaveCount":36}}
11:07:23.514 RSL: STATUS2 = {"StatusFWR":{"Version":"14.1.0.4(5bac2a2-zbbrdgpro)","BuildDateTime":"2024-08-11T17:33:02","Core":"3_0_4","SDK":"5.1.4.240801","CpuFrequency":160,"Hardware":"ESP32-D0WD-V3 v3.1","CR":"421/699"}}
11:07:23.518 RSL: STATUS3 = {"StatusLOG":{"SerialLog":2,"WebLog":2,"MqttLog":0,"SysLog":0,"LogHost":"","LogPort":514,"SSId":["netlr",""],"TelePeriod":10,"Resolution":"558180C0","SetOption":["00008009","2805C80001000600003C5A0A192800000000","00000080","00006082","00004000","00000000"]}}
11:07:23.529 RSL: STATUS4 = {"StatusMEM":{"ProgramSize":1574,"Free":281,"Heap":155,"StackLowMark":4,"PsrMax":0,"PsrFree":0,"ProgramFlashSize":4096,"FlashSize":4096,"FlashChipId":"1640C8","FlashFrequency":40,"FlashMode":"DIO","Features":["0809","8F8A87CF","00548001","000000CF","010013C0","C0000981","40024080","00200000","5400082C","00000080","00000004"],"Drivers":"1,2,!3,!4,!5,!8,9,10,12,!16,!20,!21,23,24,!27,29,!35,38,!41,50,52,!59,62,!68,82,121","Sensors":"1,2,3,5,6,127","I2CDriver":"7"}}
11:07:23.547 RSL: STATUS7 = {"StatusTIM":{"UTC":"2024-08-13T14:07:23Z","Local":"2024-08-13T11:07:23","StartDST":"2024-03-31T02:00:00","EndDST":"2024-10-27T03:00:00","Timezone":"-03:00","Sunrise":"01:41","Sunset":"16:08"}}
```
- [ ] Set `weblog` to 4 and then, when you experience your issue, provide the output of the Console log:
```lua
  Console output here:


```

### TO REPRODUCE
As problem description.

### EXPECTED BEHAVIOUR
It was expected when typing ZBINFO all the sensors information to be gathered.

### SCREENSHOTS
_If applicable, add screenshots to help explain your problem._

### ADDITIONAL CONTEXT
_Add any other context about the problem here._

**(Please, remember to close the issue when the problem has been addressed)**

**Comment by rp2019433** [comment]
Just to remark the importance of ZBINFO brings all the attributes from a sensor: when someone is using Tasmota Zigbee without any external home automation software, it's possible to gather sensors informations directly from Zigbee Bridge using Tasmota API, like this: `http://192.168.20.55/cm?cmnd=zbinfo`

In this case, it's importante all the sensors states are available with ZBINFO, or other way, so the user can have access directly to these values without any other intermediate system.

**Comment by sfromis** [comment]
Don't open an "issue" for an enhancement suggestion, especially not when there already is a thread about the topic.
https://github.com/arendst/Tasmota/discussions/20861
